cmd/root.go

```diff
@@ -332,12 +332,13 @@ Configuration
 
   By default, the Proxy will wait up to the maximum time for the startup
   endpoint to respond. The wait command requires that the Proxy be started in
-  another process with the HTTP health check enabled. If an alternate health
+  another process with the HTTP health check or prometheus enabled. If an alternate health
   check port or address is used, as in:
```

Comment:
nit: keep line length under 80 chars and capitalize Prometheus

```suggestion
  another process with the HTTP health check or Prometheus enabled. If an
  alternate health check port or address is used, as in:
```